src/ResponseFactory.php

```diff
@@ -9,10 +9,15 @@
 
 class ResponseFactory
 {
-    protected $rootView = 'app';
+    protected $rootView;
     protected $sharedProps = [];
     protected $version = null;
 
+    public function __construct()
+    {
+        $this->rootView = config('inertia.root_template');
```

Comment:
Maybe add a default value of ```'app'``` in the config function for when the config does not exist. Like:

```
$this->rootView = config('inertia.root_template', 'app');
```